Source/SwiftLintFramework/Rules/Lint/DiscardedNotificationCenterObserverRule.swift

```diff
@@ -51,6 +61,11 @@ public struct DiscardedNotificationCenterObserverRule: ASTRule, ConfigurationPro
                 return []
         }
 
+        if let lastMatch = regex("\\b[^\\(]+").matches(in: file.contents, options: [], range: range).last?.range,
```

Comment:
won't this cause false negatives? e.g.

`print(nc.addObserver(forName: .NSSystemTimeZoneDidChange, object: nil, queue: nil, using: { }))`